optaplanner-core/src/main/java/org/optaplanner/core/impl/io/XmlUnmarshallingException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.core.impl.io;
+
+public class XmlUnmarshallingException extends RuntimeException {
```

Comment:
Are we sure JAXB doesn't already have an exception that could be used?

Reply:
Sure it does: JAXBException [1], which is checked. Also, I want to avoid leaking the JAXB-specific code in OptaPlanner codebase.

[1] https://docs.oracle.com/javase/8/docs/api/javax/xml/bind/JAXBException.html

Reply:
If it leaks too deep, I see. If it doesn't, I'd rather we use it directly.

Reply:
It would leak to classes that read benchmark config, solver config, and benchmark report. And all corresponding tests. I prefer keeping the translated exception.